test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,58 @@
+require "test_helper"
+
+describe "RecipesController" do
+  describe "root" do
+    it "shows the root page" do
+      get root_path
+      must_respond_with :success
+    end
+  end
+
+  describe "index" do
+    it "returns a list of search recipes results" do
+      query = "rabbit"
+      VCR.use_cassette("recipes") do
+        RecipeSearchWrapper.search_recipes(query)
+
+        get query_recipe_path(query: query)
```

Comment:
You are both calling the lib class directly and making an HTTP request in these tests. You should split these out: controller tests make HTTP requests, and lib tests call the search wrapper.